From 75946d98725d7512b5d08b519f5a55bf9fecdf46 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Wed, 2 Sep 2026 17:16:11 +0800
Subject: [PATCH 2/2] FEDORA: do not enter autovenv by default

We have everything ready on the system end and libclang detection
consistently fails in venv.

Broken since 1.28.0.

Link: https://src.fedoraproject.org/rpms/mupdf/c/18f3d212ebb02c8a1a88b967eb1f96a87e7748da
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 scripts/wrap/__main__.py | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/scripts/wrap/__main__.py b/scripts/wrap/__main__.py
index 4440a70fe..0b54ff2e0 100644
--- a/scripts/wrap/__main__.py
+++ b/scripts/wrap/__main__.py
@@ -924,13 +924,7 @@ try:
 finally:
     del sys.path[0]
 
-autovenv.enter()
 
-# Install/import pipcl. We don't pass 'pipcl' to above call to autovenv.enter()
-# because this would do `pip install pipcl`, which seems to fail if we are
-# being run via `pip install`. It looks like `pip install` deliberately does
-# not install pip in a build venv.
-#
 try:
     import pipcl
 except ImportError:
-- 
2.55.0

